components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -85,6 +85,50 @@ A [cookbook]({{< relref "cookbooks" >}}) is the fundamental unit of configuratio
 
 [Roles]({{< relref "roles" >}}) let you define patterns and processes that exist across nodes in an organization as belonging to a single job function. Each role consists of zero (or more) attributes and a run-list.
 
+The Chef Infra Server UI lets you:
+
+- Create a Role
+- List out all the roles at one place.
+- Search for a specific role from a list of roles.
+- View the details of roles:
+  - Run List
+  - Attributes
+- Delete a Role.
```

Comment:
Use sentence punctuation consistently. Don't capitalize roles unless it is the first word in a sentence.